common/src/main/java/org/opensearch/ml/common/transport/sync/MLSyncUpNodeResponse.java

```diff
@@ -38,6 +40,7 @@ public MLSyncUpNodeResponse(StreamInput in) throws IOException {
         this.deployedModelIds = in.readOptionalStringArray();
         this.runningDeployModelIds = in.readOptionalStringArray();
         this.runningDeployModelTaskIds = in.readOptionalStringArray();
+        this.expiredModelIds = in.readOptionalStringArray();
```

Comment:
Add BWC version, ask @b4sjoo who knows the best

Reply:
will do